lib/sharezone_widgets/lib/src/svg.dart

```diff
@@ -24,7 +24,8 @@ class PlatformSvg {
       width: width,
       height: height,
       fit: fit,
-      color: color,
```

Comment:
Why is the `PlatformSvg` class even needed if we just forward to `SvgPicture` anyways? Couldn't we just remove `PlatformSvg`? @nilsreichardt  

(Should I do it?)